tests/misc_env/push_cosbench_workload.py

```diff
@@ -160,9 +160,22 @@ def push_workload(controller, client, workload_file_name):
     LOG.info(out)
     wid = out.strip().split(": ")[1]
 
+    sleep_time = 300
+    stored_bytes_prev = utils.get_utilized_space(client)
     while "PROCESSING" in get_workload_status(controller, wid):
-        LOG.info("sleeping for 30 seconds as the workload is in progress")
-        time.sleep(30)
+        LOG.info(f"sleeping for {sleep_time} seconds as the workload is in progress")
```

Comment:
@hmaheswa Instead of waiting 5 mins directly, could you please make it into retries with maximum wait to 5 mins